### PR TITLE
Added controllers and some features

### PR DESCRIPTION
**Pull request by Zawgor24**

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  def show; end
+
+  def new
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params)
+
+    if @category.save!
```
Comment (IlyaUmanets) [inline app/controllers/categories_controller.rb]: just `save`. Read more about `save vs save!`

File: app/views/devise/confirmations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (IlyaUmanets) [inline app/views/devise/confirmations/new.html.erb]: I think we need to use `slim`

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]

File: app/controllers/categories_controller.rb (hunk 1 above)
Comment (Zawgor24) [reply app/controllers/categories_controller.rb]: Ok, `save!` removed.

File: app/views/devise/confirmations/new.html.erb (hunk 2 above)
Comment (Zawgor24) [reply app/views/devise/confirmations/new.html.erb]: Changed all `erb` format files to `slim`.

File: config/routes.rb [hunk 3]
```diff
@@ -6,11 +6,20 @@
   devise_for :users
 
   resources :users, shallow: true, except: [:index] do
-    resources :posts, except: [:index]
-    resources :articles, except: [:index]
+    resources :posts, except: [:index] do
```
Comment (IlyaUmanets) [inline config/routes.rb]: No need to use brackets if you pass only one action

File: spec/controllers/articles_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```
Comment (IlyaUmanets) [inline spec/controllers/articles_controller_spec.rb]: No need to push empty specs

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 3 above)
Comment (Zawgor24) [reply config/routes.rb]: Got it, deleted brackets.

File: spec/controllers/articles_controller_spec.rb (hunk 4 above)
Comment (Zawgor24) [reply spec/controllers/articles_controller_spec.rb]: Deleted all empty specs.

**Review by IlyaUmanets — APPROVED** [review]

**Pull request merged**